src/App.tsx

```diff
@@ -14,11 +15,130 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+const App: React.FC = () => {
+  const goodsList = [...goodsFromServer];
+  const [initialButtonVisible, start] = useState(true);
+  const [isReversed, reverse] = useState(false);
```

Comment:
Follow this convension:
`const [state, setState] = useState("");`
```suggestion
  const [isReversed, setIsReversed] = useState(false);
```